client/app/routes/discussions_.$id.tsx

```diff
@@ -80,6 +82,17 @@ export const action = async ({ request, params }: ActionFunctionArgs) => {
 			const data = await request.json();
 
 			await createPost(data, userId);
+
+			const user = await prisma.user.findFirst({ where: { id: userId } });
+
+			const summary = data?.content.substring(0, 30);
+
+			await createPostNotification({
+				message: `${user?.username} posted on ${summary}}`,
```

Comment:
This wording needs improvement.

```suggestion
				message: `${user?.username} commented on ${summary}}`,
```